seed/challenges/basic-bonfires.json

```diff
@@ -568,7 +568,7 @@
       "title": "Where art thou",
       "difficulty": "1.55",
       "description": [
-        "Make a function that looks through an array (first argument) and returns an array of all objects that have equivalent property values (second argument).",
+        "Make a function that looks through an array (first argument) and returns an array of all objects that have equivalent property and value pair (second argument).",
```

Comment:
@benmcmahon100 this one line change is enough to enforce what's asked and expected of the challenge.